v2/worker/Dockerfile

```diff
@@ -1,33 +1,44 @@
-FROM node:16.11.0-bullseye-slim
-ARG VERSION
-ENV WORKER_VERSION=${VERSION}
+FROM --platform=$BUILDPLATFORM debian:bullseye-slim
```

Comment:
We use Debian for the first few steps because the inline scripts we run only work in bash. Alpine doesn't have bash. I tried hard to make the scripts POSIX-compliant so they'd work in any shell, but I just couldn't nail it.

Reply:
Also, this stage always runs on the native architecture of the build server.